tests/rkt_fly_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build fly
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/coreos/rkt/tests/testutils"
+)
+
+func TestFlyNetns(t *testing.T) {
+	testFlyNetns.Execute(t)
+}
+
+var testFlyNetns = testutils.TestFunc(func(t *testing.T) {
```

Comment:
If you write a test that is not reusable across other flavors, then I'd leave out the `TestFunc` jazz and write a plain old golang test instead.

The `Test`/`TestFunc` stuff is only interesting for reuse in other flavors.